<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shpMergeDialog</class>
 <widget class="QDialog" name="shpMergeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vector Merge Parameter Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item>
    <widget class="CardWidget" name="CardWidget_5">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="PrimarySplitPushButton" name="selectLayerSPB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text_" stdset="0">
         <string>Select Layer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PrimaryPushButton" name="selectFilePB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Select File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PrimaryPushButton" name="selectDirPB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Select Folder</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ListView" name="ListView"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PrimaryPushButton" name="clearListViewPb">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Clear Merge List</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_6">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="BodyLabel" name="BodyLabel_5">
        <property name="text">
         <string>Select Crs(Can be Null)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LineEdit" name="targetCrsLE">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_7">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="PrimaryPushButton" name="selectMapcanvasCrsPB">
        <property name="text">
         <string>Use Current Crs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PrimarySplitPushButton" name="selectLayerCrsSPB">
        <property name="text_" stdset="0">
         <string>Select Layer Crs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PrimaryPushButton" name="selectMorePB">
        <property name="text">
         <string>Select...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="BodyLabel" name="BodyLabel_4">
        <property name="text">
         <string>Result Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LineEdit" name="resLE">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ToolButton" name="selectSaveFilePB"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="PushButton" name="runPB">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="cancelPB">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SplitPushButton</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimarySplitPushButton</class>
   <extends>SplitPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ListView</class>
   <extends>QListView</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
